src/main/webapp/pages/search/search.html

```diff
@@ -1,77 +1,19 @@
-                        <span th:text="#{search.samples.tab}">_Samples_</span>
-                        <span id="sample-count" class="badge"><i class="fa fa-spinner fa-spin search-spinner" aria-hidden="true"></i></span>
-                    </a></li>
-                <li><a data-toggle="tab" href="#project-tab">
-                    <span th:text="#{search.projects.tab}">_Projects_</span>
-                    <span id="project-count" class="badge"><i class="fa fa-spinner fa-spin search-spinner" aria-hidden="true"></i></span>
-                </a></li>
-            </ul>
-            <div class="tab-content">
-                <div id="sample-tab" class="tab-pane active">
-                    <table id="samples" class="table table-striped" cellspacing="0" width="100%">
-                        <thead>
-                            <tr>
-                                <th data-data="sampleName" th:text="#{search.samples.name}"></th>
-                                <th data-data="organism" th:text="#{search.samples.organism}"></th>
-                                <th data-data="projectName" data-orderable="false" th:text="#{search.samples.project}"></th>
-                                <th class="dt-date" data-data="createdDate" th:text="#{search.samples.created}"></th>
-                                <th class="dt-date" data-data="modifiedDate" th:text="#{search.samples.modified}"></th>
-                            </tr>
-                        </thead>
-                    </table>
-                </div>
-                <div id="project-tab" class="tab-pane">
-                    <table id="projects" class="table table-striped" cellspacing="0" width="100%">
-                        <thead>
-                            <tr>
-                                <th data-data="id" width="20" th:text="#{search.projects.id}"></th>
-                                <th data-data="name" th:text="#{search.projects.name}"></th>
-                                <th data-data="organism" th:text="#{search.projects.organism}"></th>
-                                <th data-data="samples" data-orderable="false" th:text="#{search.projects.samples}"></th>
-                                <th class="dt-date" data-data="createdDate" th:text="#{search.projects.created}"></th>
-                                <th class="dt-date" data-data="modifiedDate" th:text="#{search.projects.modified}"></th>
-                            </tr>
-                        </thead>
-                    </table>
-                </div>
-            </div>
-        </main>
-
-    </div>
+<html
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  data-layout-decorate="~{template/page}"
+>
+  <head>
+    <title th:text="#{search.title}">_Search_</title>
+    <webpacker:css entry="search" />
+  </head>
+
+  <body>
+    <div id="root" layout:fragment="page" style="height: 100%"></div>
 
```

Comment:
Can you add this so we know which file it loads up?

      <!--
               This is a DOM element for React to render the search results into.
               See  src/main/webapp/resources/js/pages/search/index.tsx
       -->